p2p/simulations/network.go

```diff
@@ -718,17 +721,102 @@ func (net *Network) Load(snap *Snapshot) error {
 			return err
 		}
 	}
+
+	// Prepare connection events counter.
+	allConnected := make(chan struct{})     // closed when all connections are established
+	eventLoopStarted := make(chan struct{}) // ensures that event loop is started before it is closed
+	done := make(chan struct{})             // ensures that the event loop goroutine is terminated
```

Comment:
I had the impression we named these channels with postfix `C`. I found this contributes to clarity. Did we stop doing this?

Reply:
I did not know that suffix C is a hard requirement. This variables are local and very close in this function, so I saw no need to suffix them. Do you insist on adding the C?

Reply:
I don't insist, but I just always do :)